### PR TITLE
Handle kernel versions with '_' before build number

### PR DESCRIPTION
**Pull request by SandakovMM**

### CODE REVIEW

File: pleskdistup/common/tests/versiontests.py
Comment (kpushkaryov) [inline pleskdistup/common/tests/versiontests.py:38]: The versions are getting convoluted. I propose to test components of `KernelVersion` separately. That is, in addition to testing string representation, also test that `major`, `minor`, etc. are correct.

**Review by kpushkaryov — CHANGES_REQUESTED** [review]

File: pleskdistup/common/tests/versiontests.py
Comment (SandakovMM) [reply pleskdistup/common/tests/versiontests.py:38]: > The versions are getting convoluted.
Yep. Unfortunately, I don't think we could do much about it.
Let's add some test here as you proposed.

File: pleskdistup/common/tests/versiontests.py
Comment (kpushkaryov) [inline pleskdistup/common/tests/versiontests.py:40]: I suggest to expand other tests in the same way later.

**Review by kpushkaryov — APPROVED** [review]

File: pleskdistup/common/tests/versiontests.py
Comment (kpushkaryov) [reply pleskdistup/common/tests/versiontests.py:40]: Please rebase before merge.

**Comment by SandakovMM** [comment]
@kpushkaryov Could you please approve once again. Changes from other commit leaked into the branch, so I have to remove it and re-base once again.

**Review by kpushkaryov — APPROVED** [review]

**Pull request merged**
